Touhou Launcher/Touhou Launcher.csproj

```diff
@@ -140,7 +141,7 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="jsonnet\Newtonsoft.Json.dll" />
+    <EmbeddedResource Include="..\packages\Newtonsoft.Json.13.0.3\lib\net45\Newtonsoft.Json.dll" />

```

Comment:
Since you're using the resource manager to embed the assemblies now, why are they included here again? I removed this and the executable ran perfectly fine without the library or ballooning the executable size.